de.dlr.sc.virsat.model.extension.cefx/concept/concept.concept

```diff
@@ -123,32 +123,32 @@ Concept de.dlr.sc.virsat.model.extension.cefx
 		Applicable For [de.dlr.sc.virsat.model.extension.ps.ElementDefinition, de.dlr.sc.virsat.model.extension.ps.ElementConfiguration, de.dlr.sc.virsat.model.extension.ps.ElementOccurence, de.dlr.sc.virsat.model.extension.ps.ElementRealization];
 		Cardinality 1;
 				
-		Type massPerUnit of Category Parameter quantityKind "Mass" unit "Kilogram";
-		Type massTotal of Category Parameter quantityKind "Mass" unit "Kilogram";
+		Type mass of Category Parameter quantityKind "Mass" unit "Kilogram" description "Mass of the unit.";
+		Type massTotal of Category Parameter quantityKind "Mass" unit "Kilogram" description "Mass of the unit including the mass of the child elements.";
 		Type massTotalWithMargin of Category Parameter quantityKind "Mass" unit "Kilogram";
 		
-		Ref: massTotal = massPerUnit + summary{massTotal, 1};
-		Ref: massTotalWithMargin = massPerUnit + (massPerUnit * EquipmentParameters.marginMaturity) + summary{massTotalWithMargin, 1}; 
+		Ref: massTotal = mass + summary{massTotal, 1};
+		Ref: massTotalWithMargin = mass + (mass * EquipmentParameters.marginMaturity) + summary{massTotalWithMargin, 1}; 
 	}
 	
 	Category EquipmentPowerParameters {
 		Applicable For [de.dlr.sc.virsat.model.extension.ps.ElementDefinition, de.dlr.sc.virsat.model.extension.ps.ElementConfiguration, de.dlr.sc.virsat.model.extension.ps.ElementOccurence, de.dlr.sc.virsat.model.extension.ps.ElementRealization];
 		Cardinality 1;
 
-		Type powerUnitActive of Category Parameter quantityKind "Dimensionless" unit "No Unit" description "Should be 1 if the equipment is active in the given mode, 0 in case it is totally off."; 
+		Type powerUnitIsActive of Category Parameter quantityKind "Dimensionless" unit "No Unit" description "Should be 1 if the equipment is active in the given mode, 0 in case it is totally off."; 

```

Comment:
It still dosn't have a default value, does it?

Reply:
After I talked with @franzTobiasDLR we will flip the `PowerUnitIsActive` to `PowerUnitIsInactive` for the time being because adding a default keyword would result in a lot of work.